envconfig.go

```diff
@@ -28,8 +29,17 @@ type envTagOption struct {
 // ErrNotSet is the error that gets wrapped when a "required" env-var is not set.
 var ErrNotSet = errors.New("is not set")
 
+var tagDefaultRx = regexp.MustCompile(`^(.+),\w*(default=.*)$`)
```

Comment:
Reading this, I assume you thought `\w` is whitespace. But `\w` is "word characters" (`[0-9A-Za-z_]`); `\s` is white**s**pace. https://github.com/google/re2/wiki/Syntax